.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "libs/imagesloaded"]
```

Comment:
We should make sure this doesn't break the What's New tooltip. I _think_ the imagesLoaded instance we set up in textioMain covers that, but what I don't fully understand about jQuery plugins could just about fit in the Grand Canyon.